### ISSUE TITLE
RGPD breaks everything

### ISSUE THREAD

**Issue by erdnaxeli** [lukebarnard1/journal#76]
Because of the RGPD, the bot needs to give its consent to matrix.org.

Joining a room fail with `errcode: M_CONSENT_NOT_GIVEN`.

**Comment by ptman** [comment]
GDPR?

**Comment by ZerataX** [comment]
Yeah you have to look in the console log and click the consent link so that your matrix guest account can join rooms.
Ideally you'd only join a room tho when you actually want to comment? matrix allows you to peek into rooms without joining and that also doesn't require you to consent to gdpr.
